### PR TITLE
[CHORE] Use nest.land std instead of Deno std

### LINKED ISSUE #38: [FEATURE] Use nest.land std instead of deno.land std
We now have the std gallery, so why not use it in our products

### PR DESCRIPTION
**Pull request by johnletey**
Closes #38.

### CODE REVIEW

**Comment by johnletey** [comment]
Hmm 🤔 Why isn't the CI running?

**Comment by littledivy** [comment]
@jletey Can you do an empty commit to trigger the CI again?

**Comment by oganexon** [comment]
Maybe you need to pull the changes on your fork

**Review by SteelAlloy — APPROVED** [review]
LGTM

File: benchmarks/commands/update/deps.ts [hunk 1]
```diff
@@ -2,16 +2,16 @@
...
-import * as denoSync from "https://deno.land/std@v0.55.0/async/mod.ts";
-import * as bytes from "https://deno.land/std@v0.55.0/bytes/mod.ts";
-import * as dateTime from "https://deno.land/std@v0.55.0/datetime/mod.ts";
-import * as colors from "https://deno.land/std@v0.55.0/fmt/colors.ts";
-import * as sha512 from "https://deno.land/std@v0.55.0/hash/sha512.ts";
-import * as http from "https://deno.land/std@v0.55.0/http/mod.ts";
-import * as io from "https://deno.land/std@v0.55.0/io/mod.ts";
-import * as log from "https://deno.land/std@v0.55.0/log/mod.ts";
+} from "https://x.nest.land/std@0.56.0/testing/bench.ts";
+import * as archive from "https://x.nest.land/std@0.55.0/archive/tar.ts";
```
Comment (SteelAlloy) [inline benchmarks/commands/update/deps.ts]: Wait
Do we have `x.nest.land/std@0.55.0`?
CC @MartonDev

**Review by SteelAlloy — COMMENTED** [review]
I'm just gonna wait a little bit for confirmation from @MartonDev

File: benchmarks/commands/update/deps.ts (hunk 1 above)
Comment (martonlederer) [reply benchmarks/commands/update/deps.ts]: No, sorry we only have the latest

**Review by SteelAlloy — CHANGES_REQUESTED** [review]
We'll have to wait until previous versions of std are released.
CC @t8

**Comment by johnletey** [comment]
@MartonDev @oganexon Can we just update to the latest version? Or are there breaking changes?

But I think long term it's crucial that we have old versions published as well.

**Comment by martonlederer** [comment]
I don't know, we should ask @t8 if he's planning on release earlier versions for backwards compatibility

**Comment by t8** [comment]
I haven't yet considered doing this, although it would be possible. We'd have to make these releases manually, *unless* somebody wants to write a quick workflow script for that! 

I'm perfectly fine doing this manually though. Just let me know how many versions back we should support, as I don't know if it would make much sense to publish every single previous std version.

**Comment by t8** [comment]
I should also note that we're going to need an additional cron script to automatically publish the new std versions (if they are released). After talking with Luca, Casper, and @maximousblk, I think this will be our best route for now, without having to bother asking Deno for a webhook.

**Comment by martonlederer** [comment]
Yes, we should definitely talk about that

**Comment by SteelAlloy** [comment]
@jletey You can update to the latest version, it won't break anything.
Nevertheless, it allowed us to discuss about the std modules, thank you!

**Review by SteelAlloy — APPROVED** [review]
Great !

**Pull request merged**